src/schemas/templates.json

```diff
@@ -5,21 +5,47 @@
     "properties": {
         "templates": {
             "type": "object",
+            "description": "Holds configurations for all SAM Templates within the Workspace Folder.",
             "patternProperties": {
                 ".*": {
                     "type": "object",
                     "properties": {
                         "parameterOverrides": {
                             "type": "object",
+                            "default": {},
+                            "description": "Configures the SAM Template's parameter values used when deploying the template.",
                             "patternProperties": {
                                 ".*": {
                                     "type": "string"
                                 }
                             }
+                        },
+                        "handlers": {
+                            "type": "object",
+                            "default": {},
+                            "description": "Holds configurations for each Lambda function contained in this SAM Template.",
+                            "patternProperties": {
+                                ".*": {
+                                    "type": "object",
+                                    "description": "Configuration for a Lambda handler within a SAM Template.",
+                                    "properties": {
+                                        "event": {
+                                            "type": "object",
+                                            "default": {},
+                                            "description": "Event data passed to the Lambda function when it is invoked locally. If you don't specify this option, an empty payload is used as a default."
+                                        },
+                                        "environmentVariables": {
+                                            "type": "object",
+                                            "default": {},
+                                            "description": "Environment variables passed to the Lambda function when it is run locally."
```

Comment:
Maybe add a note that environment variables need to be present in the SAM template before they can be overridden.